### ISSUE TITLE
Description of send command

### ISSUE THREAD

**Issue by ramsom** [ssjoholm/rfxcmd#5]
Hello.

Its not really a issue but more a question. Im trying to use AC (under signal printout 0×11 – Lightning2) command and i suppose i need to use rfxsend.py. Is there an example on how to use this in this scenario?

Thanks

**Comment by ssjoholm** [comment]
Hi,

The RFXSend.py is very simple, as it can only send the raw data towards RFXtrx. It will use the RFXcmd which need to have the "Socket Server" activated (config.xml). The best way to find out the raw command are usually to use the RFXManager and send the command from there first.

**Comment by ramsom** [comment]
Could you give an example with the RFXSend.py on the recorded listen response below:

Received                = 0B 11 00 06 00 BF F4 96 0A 01 0F 60
Date/Time               = 2016-01-22 23:08:51
Packet Length           = 0B
Packettype              = Lighting2
Subtype                 = AC
Seqnbr                  = 06
Id                      = 00BFF496
Unitcode                = 10
Command                 = On
Dim level               = 100%
Signal level            = 6

will be much appreciated.

**Comment by Jantje2000** [comment]
Indeed. My question is: What is the raw data. The socketserver etc. can go, but I don't know wich meesage I have to send
